addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -96,6 +120,7 @@
                 <field name="legend_blocked" invisible="1"/>
                 <field name="legend_done" invisible="1"/>
                 <div class="oe_title">
+                    <span class="o_form_label"><field name="name"  readonly="1" nolabel="1"/></span>
```

Comment:
I don't think specifying the same field multiple times is a good idea.
Why do you need this bit of code ?